src/mame/namco/namcos10.cpp

```diff
@@ -512,6 +684,7 @@ class namcos10_state : public driver_device
 	optional_device<ns10_decrypter_device> m_decrypter; // not every game has a decrypter implemented yet, so optional for now
 
 	unscramble_func m_unscrambler;
+	std::function<void()> m_psx_remapper;
```

Comment:
If you turn this into a `timer_expired_delegate` then you can pass it to `synchronize` directly without an extra trampoline.

Reply:
Just to make sure I'm not misunderstanding or doing something wrong, should lambdas work with that method or should I turn the remap functions into named functions instead of using a lambda function? I gave it a go using lambdas but it always ends up giving a signature error (`lambda [](s32)->void` vs `void(s32)`).